spack/packages/dla-future/package.py

```diff
@@ -32,7 +32,7 @@ class DlaFuture(CMakePackage, CudaPackage):
     # https://github.com/eth-cscs/DLA-Future/issues/420
     conflicts("umpire@6:")
 
-    depends_on("hpx cxxstd=14 networking=none +async_mpi")
+    depends_on("hpx cxxstd=17 networking=none +async_mpi")
```

Comment:
Same here, HPX 1.8.0 will already require C++17, so `cxxstd=17` can be left out to allow it to be compiled with C++20 as well.